src/core_modules/capture-core/components/Notes/Notes.component.js

```diff
@@ -199,7 +199,7 @@ const NotesPlain = ({
                                     </div>
                                     <div className={classes.noteItemDate} data-test="note-date">
                                         <span>
-                                            <Tooltip content={convertClientToList(fromServerDate(n.storedAt), dataElementTypes.DATETIME)}>
+                                            <Tooltip content={convertClientToList(moment(fromServerDate(n.storedAt).getClientZonedISOString()).toISOString(), dataElementTypes.DATETIME)}>
```

Comment:
This change ensures that the date is in client format when being passed to convertClientToList (should include timezone-info).